src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react';
+
+interface Props {
+  errorMessage: string;
+  onClose: () => void;
+}
+
+export const Notification: React.FC<Props> = ({ errorMessage, onClose }) => {
+  const [showNotification, setShowNotification] = useState(false);
+
+  useEffect(() => {
+    let hideTimeout: NodeJS.Timeout;
+
+    if (errorMessage) {
+      setShowNotification(true);
+
+      hideTimeout = setTimeout(() => {
+        setShowNotification(false);
+        onClose();
+      }, 3000);
+    }
+
+    return () => {
+      clearTimeout(hideTimeout);
+      setShowNotification(false);
+    };
+  }, [errorMessage, onClose]);
+
+  const handleClose = () => {
+    setShowNotification(false);
+    onClose();
+  };
+
+  return (
+    <div
+      className={`notification is-danger is-light has-text-weight-normal ${showNotification
+        ? ''
+        : 'hidden'}`}
```

Comment:
it's better to use classnames library for such cases

Reply:
not fixed. feel free to write in the chat if you have questions
<img width="403" alt="image" src="https://github.com/mate-academy/react_todo-app-loading-todos/assets/98764641/23696f5f-dd30-4139-ae44-c1abb3e60111">